jmh-tests/src/main/java/org/eclipse/collections/impl/jmh/FlatCollectTest.java

```diff
@@ -38,26 +38,30 @@
     private final MutableList<MutableList<Integer>> integersEC = FastList.newWithNValues(COUNT, () -> Interval.oneTo(LIST_SIZE).toList());
 
     @Benchmark
-    public void serial_lazy_jdk()
+    public List<Integer> serial_lazy_jdk()
     {
         List<Integer> flatMap = this.integersJDK.stream().flatMap(Collection::stream).collect(Collectors.toList());
+        return flatMap;
     }
 
     @Benchmark
-    public void serial_lazy_streams_ec()
+    public List<Integer> serial_lazy_streams_ec()
     {
         List<Integer> flatMap = this.integersEC.stream().flatMap(Collection::stream).collect(Collectors.toList());
+        return flatMap;
     }
 
     @Benchmark
-    public void serial_eager_ec()
+    public MutableList<Integer> serial_eager_ec()
     {
         MutableList<Integer> flatCollect = this.integersEC.flatCollect(e -> e);
+        return flatCollect;
     }
 
     @Benchmark
-    public void serial_lazy_ec()
+    public MutableList<Integer> serial_lazy_ec()
     {
         MutableList<Integer> flatCollect = this.integersEC.asLazy().flatCollect(e -> e).toList();
+        return flatCollect;
```

Comment:
For all four of these, we can just inline the variable, like:
```suggestion
        return this.integersEC.asLazy().flatCollect(e -> e).toList();
```